docker/importer/importer.py

```diff
@@ -37,6 +41,8 @@
 _OSS_FUZZ_EXPORT_BUCKET = 'oss-fuzz-osv-vulns'
 _EXPORT_WORKERS = 32
 _NO_UPDATE_MARKER = 'OSV-NO-UPDATE'
+_BUCKET_BATCH_SIZE = 50
+_BUCKET_THREAD_POOL = 10
```

Comment:
s/_BUCKET_THREAD_POOL/_BUCKET_THREAD_COUNT/ instead to make it clearer this is a count/number and not an actual thread pool object.